fe/fe-core/src/main/java/org/apache/doris/analysis/SelectStmt.java

```diff
@@ -1040,6 +1048,21 @@ private void analyzeAggregation(Analyzer analyzer) throws AnalysisException {
         aggExprs.clear();
         TreeNode.collect(substitutedAggs, Expr.isAggregatePredicate(), aggExprs);
 
+        if (metaQuery) {
```

Comment:
should we remove meta hint, and turn ScanNode to OlapMetaScanNode automaticaly when we found some function need to do meta query?